core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -209,6 +210,7 @@ public final class ServerBuilder implements TlsSetters {
     private boolean enableDateHeader = true;
     private Supplier<? extends RequestId> requestIdGenerator = RequestId::random;
     private Http1HeaderNaming http1HeaderNaming = Http1HeaderNaming.ofDefault();
+    private Duration unloggedExceptionReportInterval = Duration.ofSeconds(60);
```

Comment:
What would be the sensible default? I think it shouldn't take too long until a user realizes that there were exceptions, although a too short interval can be annoying. How about 10 seconds?

Reply:
Sounds great. It might be more sensible for users to view exceptions with an interval less than 60 seconds. 10 seconds might be great.